addons/pos_restaurant/static/src/overrides/models/pos_bus.js

```diff
@@ -35,30 +35,25 @@ patch(PosBus.prototype, {
     // Sync the number of orders on each table with other PoS
     // using the same floorplan.
     async ws_syncTableCount(data) {
-        const missingTable = data.find((table) => !(table.id in this.pos.tables_by_id));
+        const missingTable = data.find(
+            (table) => !(table.id in this.pos.models["restaurant.table"].getAllBy("id"))
```

Comment:
extract this `table.id in this.pos.models["restaurant.table"].getAllBy("id")` as a method in the models. Maybe name it `isPresentBy` where `isPresentBy('id', <id_value>)` returns `true` if the record with that "id" is loaded.

Reply:
Isn't it weird to create an API on the model named `isPresentBy` when we have a general API that does exactly the same thing?
`this.pos.models["restaurant.table"].get(id)`